vector-im/element-android
# Add aggregated unread indicator for spaces in the new layout

## Type of change


## Content


In the new app layout, adding a notification badge on top of the floating action button which opens the spaces list bottom sheet.

## Motivation and context


Closes #8157 

## Screenshots / GIFs



<img src="https://user-images.githubusercontent.com/46314705/220651831-1f8957ea-4532-4c4f-a3d2-d1086c18115d.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/220651839-ab587c2d-382f-46fc-ba6a-7d1de4b0412f.png" width=300 />
<img src="https://user-images.githubusercontent.com/46314705/220652177-1eef9724-3722-4996-9041-78f43209faba.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/220652207-8608c5dc-7ae3-4fdd-b949-f5fb4432e051.png" width=300 />
<img src="https://user-images.githubusercontent.com/46314705/220652537-a2397cb4-b0d0-4b0d-b356-4a3ce05c0700.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/220652545-fd030443-2307-4dba-9278-a3742864510c.png" width=300 />
<img src="https://user-images.githubusercontent.com/46314705/220652787-eb2d5150-4fe2-4553-8396-6c2b72e53924.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/220652796-babaec45-67bd-4cce-ac84-4434f21a6f08.png" width=300 />
<img src="https://user-images.githubusercontent.com/46314705/220653688-09b0ef84-84b7-4e3d-9949-60956432d0ef.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/220653699-0324953f-ea2e-4136-bb8a-79f6accb936a.png" width=300 />
<img src="https://user-images.githubusercontent.com/46314705/220653829-26cfe7e2-a9ab-4e65-9ac6-4539719c859b.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/220653834-d89c4ae3-22d2-4608-bf4c-0f8ab5c76bc2.png" width=300 />


## Tests



- Open the app with new layout
- Send message in some rooms and check the badge counter is correctly displayed
- Send invite to a space
- Check the badge counter is red while the space invite is pending

## Tested devices

- OS version(s): Android 11

## Checklist
